### DISCUSSION TITLE
Is there an option to see torrent infos with single click/tap?

### DISCUSSION THREAD

**Discussion by nizut** [VueTorrent/VueTorrent#2010] [Q&A]
I like the design of this ui, but it's little annoying to see torrent infos with double click/tap. I can't find an option does this with single click/tap. Maybe it's  unable to change on gui?

**Comment by Larsluph** [comment]
Not at the moment.

An attempt was made for simplifying the process but I don't think it will ever come out: #1880

However I don't see any big impact on switching from double-click to single-click, we might consider this as a possible improvement.
